test/src/system/installation_process_test.coffee

```diff
@@ -57,3 +57,21 @@ testGroup "Installation process with specified elements", template: "editor_with
       assert.equal editorElement.inputElement, document.getElementById("my_input")
       assert.equal editorElement.value, "<div>Hello world</div>"
       done()
+
+  testGroup "Installation process with text after closing tag", template: "editor_empty", ->
+    test "parses text after closing tag as separate block", (done) ->
+      input = document.createElement("input")
+      input.setAttribute("id", "my_input")
+      input.setAttribute("value", "<h1>Test</h1>abc")
+
+      element = document.createElement("trix-editor")
+      element.setAttribute("input", "my_input")
+
+      container = document.getElementById("trix-container")
+      container.innerHTML = ""
+      container.appendChild(input)
+      container.appendChild(element)
+
+      element.addEventListener "trix-initialize", ->
+        assert.equal element.value, "<h1>Test</h1><div>abc</div>"
+        done()
```

Comment:
I think this could be tested less verbosely in the HTML loading tests. This one is pretty similar: https://github.com/basecamp/trix/blob/92f679a207e2e31194963f857162578b47b68800/test/src/system/html_loading_test.coffee#L57-L62

Add a new test group using `template: "editor_empty"` and change the second assertion to expect no block attributes: ` assert.blockAttributes([2, 4], [])`